packages/commonwealth/client/scripts/views/components/NewThreadFormLegacy/NewThreadForm.tsx

```diff
@@ -59,7 +59,6 @@ import {
   getTextFromDelta,
   serializeDelta,
 } from '../react_quill_editor/utils';
-import ContestThreadBanner from './ContestThreadBanner';
```

Comment:
please remove ContestThreadBanner directory including tsx, scss and index files

Reply:
updated